build.sbt

```diff
@@ -107,12 +107,12 @@ lazy val `anorm-akka` = (project in file("akka"))
       "Tatami Releases".at(
         "https://raw.github.com/cchantep/tatami/master/snapshots")),
     libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-stream" % "2.4.11" % "provided",
+      "com.typesafe.akka" %% "akka-stream" % "2.4.16" % "provided",
       "org.eu.acolyte" %% "jdbc-scala" % acolyteVersion % Test
     ) ++ Seq(
       "specs2-core",
       "specs2-junit"
-    ).map("org.specs2" %% _ % "3.8.3" % Test) ++ Seq(
+    ).map("org.specs2" %% _ % "3.8.6" % Test) ++ Seq(
       "com.typesafe.akka" %% "akka-stream-contrib" % "0.3-9-gaeac7b2" % Test)
```

Comment:
akka-stream-contrib 0.6 should be ok: http://search.maven.org/#artifactdetails%7Ccom.typesafe.akka%7Cakka-stream-contrib_2.12%7C0.6%7Cjar

> If ok, Tatami resolver can be removed